tests/package/test.sc

```diff
@@ -458,5 +470,11 @@ def main(args: String*): Unit = {
       ???
   }
 
-  tests.filter { t => predicate(name(t))}.foreach(run(_))
+  val testsToRun = tests.filter { t => predicate(name(t))}
+  testsToRun.foreach(run(_))
+  val results = testsToRun.map { t => Await.result(t.result, 1.hour) }
+  if (!results.forall(_ == true)) {
```

Comment:
Since it's `"There were errors"` kind of logic may be `if (results.exists(_ == false))` would be better?

Reply:
Well, maybe is too compact, but this actually does what I want.

```
@ Nil.forall(_ == false)
res0: Boolean = true
```

Although, granted, we should probably show an error message if no tests match the provided string. But, I think this is fine for now :-)

Reply:
Should we print the errors here?